app/Models/Foundation/Summit/Speakers/PresentationSpeaker.php

```diff
@@ -1403,7 +1416,9 @@ public function moderated_presentations($summit_id, $published_ones = true)
      */
     public function getPresentation($presentation_id)
     {
-        return $this->presentations->get($presentation_id);
+        $this->getAssignedPresentations()->filter(function ($element) use ($presentation_id) {
```

Comment:
please use a criteria query

Reply:
this method has been removed bc is not used